understanding/20/error-identification.html

```diff
@@ -72,21 +72,18 @@ <h2>Intent of Error Identification</h2>
          that user agents or assistive technologies can use to identify an error and provide
          error information to the user. For example, certain technologies can specify that
          the user's input must not fall outside a specific range, or that a form field is required.
-         Currently, few technologies support this kind of programmatic information, but the
-         Success Criterion does not require, nor prevent it.
-         
+         This type of programmatic information is not required for this success criterion, but is covered
+         by other criteria such as <a href="#name-role-value">4.1.2 Name, Role, Value</a>.
```

Comment:
I think this is a little too strong. Some stuff could be argued to fall under 4.1.2, but I'm not so sure that 4.1.2 required that for example an input range has to be expressed through a component's programmatic properties.

```suggestion
         This type of programmatic information is not required for this success criterion, but may be covered
         by other criteria such as <a href="#name-role-value">4.1.2 Name, Role, Value</a>.
```

Reply:
makes sense, yes